### PR TITLE
Remove un-needed logic from supplier index page

### PR DESCRIPTION
**Pull request by bmrsny**
Removed some logic from the supplier show page that was not supposed to be there.

### CODE REVIEW

**Comment by bmrsny** [comment]
Thanks for the help Alex

File: app/views/suppliers/show.html.erb [hunk 1]
```diff
@@ -1,21 +1,4 @@
...
-				<%= image_tag item.image.url(:medium) %>
-				<% if !item.retired %>
-				<%= button_to "Add to Cart", cart_index_path(item_id: item), class: "btn btn-lg btn-default" %>
-				<% end %>
-			</li>
-			<% end  %>
-		</ul>
-	<% end %>
+	<%= render partial: "./partials/item_index" %>
 </div>
```
Comment (scottalexandra) [inline app/views/suppliers/show.html.erb:4]: Glad we changed this.

File: app/views/suppliers/show.html.erb [hunk 2]
```diff
@@ -1,21 +1,4 @@
...
-				Quantity:	<%= item.quantity %></br>
-				<%= image_tag item.image.url(:medium) %>
-				<% if !item.retired %>
-				<%= button_to "Add to Cart", cart_index_path(item_id: item), class: "btn btn-lg btn-default" %>
-				<% end %>
-			</li>
-			<% end  %>
-		</ul>
-	<% end %>
+	<%= render partial: "./partials/item_index" %>
```
Comment (scottalexandra) [inline app/views/suppliers/show.html.erb:3]: This page will most likely change based off of the wire frames laura is doing

**Comment by scottalexandra** [comment]
Looks Good!

**Pull request merged**